Sources/Commands/SwiftTestCommand.swift

```diff
@@ -75,23 +75,39 @@ struct SharedOptions: ParsableArguments {
     /// to choose from (usually in multiroot packages).
     @Option(help: .hidden)
     var testProduct: String?
+}
 
-    /// Whether to enable support for XCTest.
+/// Which testing libraries to use (and any related options.)
+struct TestLibraryOptions: ParsableArguments {
```

Comment:
This type belongs to `CoreCommands/Options.swift` or (ideally) a separate file. `SwiftTestCommand.swift` is already too big and hard to navigate, we should avoid adding more types or extensions to it.

Reply:
I will move it.